scipy/stats/_morestats.py

```diff
@@ -3581,7 +3581,7 @@ def wilcoxon(x, y=None, zero_method="wilcox", correction=False,
     The presence of "ties" (i.e. not all elements of ``d`` are unique) or
```

Comment:
The bit above looks like it needs to be updated since `method='auto'` now selects `PermutationMethod` sometimes.